lib-dmtf/model/Chassis_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/ODIM-Project/ODIM/lib-utilities/config"
 )
 
-var chassisJSON = "{\"@odata.context\":\"/redfish/v1/$metadata#Chassis.Chassis\",\"@odata.id\":\"/redfish/v1/Chassis/1/\",\"@odata.type\":\"#Chassis.v1_6_0.Chassis\",\"@odata.etag\":\"W/\\\"ACCE5EFE\\\"\",\"Id\":\"1\",\"Description\":\"\",\"Name\":\"Computer System Chassis\",\"AssetTag\":\"\",\"ChassisType\":\"RackMount\",\"DepthMm\":0,\"EnvironmentalClass\":\"\",\"HeightMm\":0,\"IndicatorLED\":\"On\",\"Manufacturer\":\"HPE\",\"Model\":\"ProLiant DL380 Gen10\",\"PartNumber\":\"\",\"PowerState\":\"\",\"SerialNumber\":\"2M291101JZ\",\"SKU\":\"868704-B21\",\"UUID\":\"\",\"WeightKg\":0,\"WidthMm\":0,\"Links\":{\"ComputerSystems\":[{\"@odata.id\":\"/redfish/v1/Systems/1/\"}],\"ManagedBy\":[{\"@odata.id\":\"/redfish/v1/Managers/1/\"}]},\"Location\":{\"@odata.id\":\"\"},\"LogServices\":{\"@odata.id\":\"\"},\"Assembly\":{\"@odata.id\":\"\"},\"NetworkAdapters\":{\"@odata.id\":\"/redfish/v1/Chassis/1/NetworkAdapters/\"},\"PCIeSlots\":{\"@odata.id\":\"\"},\"PhysicalSecurity\":{\"IntrusionSensor\":\"\",\"IntrusionSensorNumber\":0,\"IntrusionSensorReArm\":\"\"},\"Power\":{\"@odata.id\":\"/redfish/v1/Chassis/1/Power/\"},\"Sensors\":{\"@odata.id\":\"\"},\"Status\":{\"Health\":\"OK\",\"State\":\"Starting\",\"Oem\":{}},\"Thermal\":{\"@odata.id\":\"/redfish/v1/Chassis/1/Thermal/\"}}"
+var chassisJSON = "{\"@odata.context\":\"/redfish/v1/$metadata#Chassis.Chassis\",\"@odata.id\":\"/redfish/v1/Chassis/1/\",\"@odata.type\":\"#Chassis.v1_16_0.Chassis\",\"@odata.etag\":\"W/\\\"ACCE5EFE\\\"\",\"Id\":\"1\",\"Description\":\"\",\"Name\":\"Computer System Chassis\",\"AssetTag\":\"\",\"ChassisType\":\"RackMount\",\"DepthMm\":0,\"EnvironmentalClass\":\"\",\"HeightMm\":0,\"IndicatorLED\":\"On\",\"Manufacturer\":\"HPE\",\"Model\":\"ProLiant DL380 Gen10\",\"PartNumber\":\"\",\"PowerState\":\"\",\"SerialNumber\":\"2M291101JZ\",\"SKU\":\"868704-B21\",\"UUID\":\"\",\"WeightKg\":0,\"WidthMm\":0,\"Links\":{\"ComputerSystems\":[{\"@odata.id\":\"/redfish/v1/Systems/1/\"}],\"ManagedBy\":[{\"@odata.id\":\"/redfish/v1/Managers/1/\"}]},\"Location\":{\"@odata.id\":\"\"},\"LogServices\":{\"@odata.id\":\"\"},\"Assembly\":{\"@odata.id\":\"\"},\"NetworkAdapters\":{\"@odata.id\":\"/redfish/v1/Chassis/1/NetworkAdapters/\"},\"PCIeSlots\":{\"@odata.id\":\"\"},\"PhysicalSecurity\":{\"IntrusionSensor\":\"\",\"IntrusionSensorNumber\":0,\"IntrusionSensorReArm\":\"\"},\"Power\":{\"@odata.id\":\"/redfish/v1/Chassis/1/Power/\"},\"Sensors\":{\"@odata.id\":\"\"},\"Status\":{\"Health\":\"OK\",\"State\":\"Starting\",\"Oem\":{}},\"Thermal\":{\"@odata.id\":\"/redfish/v1/Chassis/1/Thermal/\"}}"
```

Comment:
Please replace version with the string constant defined

Reply:
this file is no longer used by odimra